### PR TITLE
Fix failing npm install. Updated dependencies.

### PR DESCRIPTION
**Pull request by maxrandolph**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,15 +3,14 @@
   "version": "2.0.0",
   "description": "Connect to web data from Tableau.",
   "scripts": {
-    "corsproxy": "export CORSPROXY_PORT=8889 || set CORSPROXY_PORT=8889 && node ./Simulator/utils/corsproxy.js",
```
Comment (chenteng2016) [inline package.json:6]: This is the migrated cors-proxy that could be run with npm start already, did you accidentally remove this?

File: Simulator/components/AddressBar.jsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
```
Comment (chenteng2016) [inline Simulator/components/AddressBar.jsx:1]: If we have .jsx changes, do we need to do bundle and wdc-simulator.js gets its updates?

File: .npmrc [hunk 3]
```diff
@@ -0,0 +1,2 @@
+; enable npm install to succeed by forcing the old method of peer dependency handling
+legacy-peer-deps=true
```
Comment (chenteng2016) [inline .npmrc:2]: nice change for letting different versions of npm work. (npm install --legacy-peer-deps will also work for such npm versions)

File: Simulator/components/AddressBar.jsx (hunk 2 above)
Comment (maxrandolph) [reply Simulator/components/AddressBar.jsx:1]: Changing the `prop-types` import to this style will allow support without breaking the build should we upgrade the version of `react` to > 15 in the future. I've updated `wdc-simulator.js` with `npm run bundle` and added to the PR.

File: package.json (hunk 1 above)
Comment (maxrandolph) [reply package.json]: Yes, I was initially looking at another PR that avoided using `corsproxy`, but I reverted this change for inclusion :)

**Review by chenteng2016 — APPROVED** [review]

**Pull request merged**